lib/brewdler/utils.rb

```diff
@@ -1,10 +1,29 @@
 module Brewdler
   def self.system cmd, *args
-    pid = fork do
-      $stdout.reopen("/dev/null")
-      exec(cmd, *args)
+    verbose = ARGV.verbose?
+    rd, wr = IO.pipe
+    logs = []
+    begin
+      pid = fork do
+        rd.close
+        $stdout.reopen(wr)
+        wr.close
+        exec(cmd, *args)
+      end
+
+      wr.close
+      while buf = rd.gets
+        puts buf if verbose
+        logs << buf
+      end
+    rescue
+      rd.close
     end
```

Comment:
I don't want to block this being included but https://github.com/mikemcquaid/HookHand/blob/028a138c0b8f76e9e3baf7a71619b069755d3b5e/hookhand.rb#L137-L160 should show a use of `IO.popen` which will hopefully clarify things. If it's still too confusing feel free to say so and we can just merge this as-is.